docs/articles/posts/20241203_RISE_article.md

```diff
@@ -0,0 +1,301 @@
+
+### Did you find this tool useful and are there any improvements you’d like to see?
+
+The headline for this section all users found this tool to be very useful!
+
+That being said, we received various suggestions for improvements. Users without labels in the tool asked for the following:
+
+1. Make the prediction of thumbnail images on the right clearer.
+
+2. Double click images to view them larger and in more detail.
+
+3. Include a way to flag the image as 'questionable'.
+
+4. Make it easier to identify which specific image is a specific dot (the hover tool appears to not have worked for a lot of people!).
+
+5. Improve the training on what everything in the tool actually 'means'.
+
+And for those with labels:
+
+1. Add the capability to generate and add new images within the tool.
+
+2. Add model accuracy on the tool.
+
+3. Make it 'less buggy' – this may partially be due to the tool being hosted on GitHub Codespaces.
+
+4. Highlight points when clicking the thumbnail on the right.
+
+5. Zoom in on the scatter plot.
+
+6. Have a reset button.
+
+Point 2 is interesting as it is the only piece of feedback specific to the tool when labels are included. 
+
+Overall, the tool appears to have been used successfully, with users enjoying the experience and identifying risks and misclassifications within the tool. The feedback we received from users was mixed in length and quality, which means directly comparing whether the tool is more useful with or without labels is hard. What is clear is that both cohorts were similarly successful in using the tool.
+
+I did notice that within the feedback there was some confusion regarding whether some behaviour was part of the tool or part of the AI classifier. One user suggested that the tool should have another class called 'other'. Whilst this is a good insight, it is in fact the model which would need another class. Additionally, some users mixed up terms such as efficiency and accuracy.
+
+This highlights the need for thorough staff training when using this tool, and many suggestions regarding the tools usability should be acted upon before the next iteration of this work.
+
+Additionally, none of the feedback referenced any edge-case (hybrid animal) images, instead pointing out occurrences when easily identifiable animals were performing certain behaviours or contained certain features. This is interesting, and may imply that the edge-case images we used were not found to be very helpful by the assurers. Alternatively, our guess of what an edge-case image was might have been completely wrong – the model interpreted images in a completely different way to human users.
+
+## Conclusion and Future Work 

```

Comment:
i like this section a lot, but maybe with it being a blog you could name is something more jazzy like "How successful were we?" and "What's next?"